### DISCUSSION TITLE
CPU power limits doesn't change in G Helper? TUF A15 2022 (6800H)

### DISCUSSION THREAD

**Discussion by inzork** [seerge/g-helper#690] [Q&A] [answered]
Hey, so I was wondering, is this only a visual "bug" or cpu power limits really don't change? I switch between Balanced and Turbo mode, the fan curve change works, but in both mode SPL + sPPT and fPPT stays at 125W (+ GPU doesn't get overclocked in Turbo mode like in armory crate). Sorry if it's mentioned somewhere, I couldn't find anything related to this.

**Comment by seerge** [comment, marked as answer]
@inzork hello, there is no way to "read" default power limits (except running some benchmark and looking at actual power in HWinfo), so app show slider just on some average value.

Also keep in mind, that until "apply" custom power limits are not applied, and slider does nothing. 

Given that your CPU is similar to one my G14, i would guess default limits are similar to ones listed in readme https://github.com/seerge/g-helper#rocket-performance-modes

**Reply by inzork** [reply]
Thanks for the quick reply! Ah I feel kinda stupid, somehow I completely missed the "Apply" checkbox despite it being in front of me haha.
